boards/xtensa/yd_esp32/yd_esp32.dts

```diff
@@ -0,0 +1,182 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32/esp32_wroom_32ue_n16.dtsi>
+#include <zephyr/dt-bindings/led/led.h>
+#include "yd_esp32-pinctrl.dtsi"
+
```

Comment:
Perhaps it is missing  
`#include <zephyr/dt-bindings/input/input-event-codes.h>`

Reply:
Fixed.

https://github.com/zephyrproject-rtos/zephyr/blob/4a7a60261390a9bdccc954db4f282e02b1ae6704/boards/xtensa/yd_esp32/yd_esp32.dts#L1-L30